common/src/main/java/com/nearsoft/fgaribay/mgmt/model/Product.java

```diff
@@ -0,0 +1,51 @@
+package com.nearsoft.fgaribay.mgmt.model;
+
+import org.hibernate.validator.constraints.NotEmpty;
+import org.hibernate.validator.constraints.Range;
+
+import java.io.Serializable;
+
+public class Product implements Serializable {
+  @Range(min = 1, max = 65535)
```

Comment:
As the business rules are undefined, I created the following scenario:
* The system must allow the following legacy requirements:
  * Some outdated equipment will make use of the product information (e.g. 2 bytes in a barcode store the product ID).
  * The IDs cannot be autogenerated because control is needed by business over the IDs.

Do you suggest I change the mock requirements to match a scenario more similar to SKT?